l3backend/l3backend-color.dtx

```diff
@@ -246,8 +246,11 @@
     \cs_gset_protected:Npn \__kernel_backend_scope_end:
       {
         \__kernel_backend_literal:n { x:grestore }
-        \__kernel_backend_literal:n
-          { pdfcolorstack ~ \g_@@_backend_stack_int current }
+        \use:x
```

Comment:
Rather than `\use:x`, we should be making an `x-type variant of `\__kernel_backend_literal:n` here and using that.

Reply:
Ah, sorry, we have the variant: just use it :)

Reply:
Ya, just found
https://github.com/latex3/latex3/blob/893c1833f8a215e81489e9be5890bc2cfe210d70/l3backend/l3backend-basics.dtx#L147-L150